README.md

```diff
@@ -2,6 +2,8 @@
 
 Automated, pre-built packages for Wayland (sway/wlroots) tools for NixOS.
 
+*NOTE:* this overlay depends on nixpkgs-unstable (> 18.09)
+
```

Comment:
Can you remove this now that I'm building against 18.09 as well? If you want, you can move the note at the bottom that mentions this, up to the top where you have this comment.